src/world/area_hos/hos_05/hos_05_6_narrator.c

```diff
@@ -2,44 +2,60 @@
 
 #include "../common/Narrator.inc.c"
 
+#if VERSION_PAL
+#define DURATION_24 (14)
```

Comment:
Should these be named? Because I have no idea how to make these sound meaningful

Reply:
We can save these for later.